pyccel/parser/semantic.py

```diff
@@ -3562,32 +3581,19 @@ def _visit_ClassDef(self, expr):
                    bounding_box=(self._current_fst_node.lineno, self._current_fst_node.col_offset),
                    severity='error')
 
-        ms = []
         for i in methods:
             self._visit_FunctionDef(i)
             m_name = i.name.replace("'", '')
```

Comment:
I suspect this is no longer needed. Do you mind testing?

Reply:
It s still used, it s here where the `__init__` method is visited and raises an error if no one found https://github.com/pyccel/pyccel/blob/0bb6b91d22218d114e61e5497db52ddc0a090167/pyccel/parser/semantic.py#L3572-L3585
and here where the rest of the methods if visited https://github.com/pyccel/pyccel/blob/0bb6b91d22218d114e61e5497db52ddc0a090167/pyccel/parser/semantic.py#L3587-L3590

Reply:
I just mean the last line which removes quotes

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1484/commits/a93c88ddfb418120ef1227bb6c42c4171a2cfb49.